tests/integration/test_password_store.py

```diff
@@ -0,0 +1,193 @@
+from kubernetes.client import CustomObjectsApi
+from kubernetes.client import CoreV1Api
+from kubernetes.client.rest import ApiException
+from .common import random_str
+import base64
+
+group = 'management.cattle.io'
+version = 'v3'
+namespace = 'local'
+plural = 'clusterloggings'
+clusterId = "local"
+globalNS = "cattle-global-data"
+
+
+def test_cluster_logging_elasticsearch(admin_mc):
+    client = admin_mc.client
+    secretPassword = random_str()
+    indexPrefix = "prefix"
+    endpoint = "https://localhost:8443/"
+    name = random_str()
+
+    es = client.create_cluster_logging(
+                                        name=name,
+                                        clusterId=clusterId,
+                                        elasticsearchConfig={
+                                            'authPassword': secretPassword,
+                                            'endpoint': endpoint,
+                                            'indexPrefix': indexPrefix})
+
+    # Test password not present in api
```

Comment:
Can we use remove_resource too?

Reply:
added remove_resource to both the objects